### ISSUE TITLE
hello

### ISSUE THREAD

**Issue by SWHL** [jaggiK/rapidocr_openvinogpu#1]
Hello, I am the developer of rapid_ocr.
Would you like to merge this project into the RapidOCR repository? 
Your work just fills the gap of openvino's reasoning on GPU.

**Comment by jaggiK** [comment]
Yes, happy to. This implementation works only for a specific resolution. Let me know how we can integrate this. Thanks

**Comment by SWHL** [comment]
just curious, why does this implementation only work for a specific resolution?
Openvino supports the dynamic inference in the `rapidocr_openvino`.

**Comment by jaggiK** [comment]
For two reasons:
1. I was able to good results with a specific resolution 
2. older version of OpenVino don’t support dynamic GPU inference. 

Latest version supports dynamic inference, I will be working on transitioning to that. This will likely to solve this requirement. 

After that, I will try to branch off from your main branch and create a PR. Does that sound good?

**Comment by SWHL** [comment]
For first reason, from my experience, the shape of inputing to the text recognition model must to be dynamic if you want a good result.


> After that, I will try to branch off from your main branch and create a PR. Does that sound  good?

It's good, I am looking forward to your PR.
